airbyte-webapp/src/locales/en.json

```diff
@@ -763,6 +763,7 @@
   "connectorBuilder.loadingStreamList": "Loading",
   "connectorBuilder.noStreamSelected": "No stream selected",
   "connectorBuilder.streamListUrlError": "Could not load URL",
+  "connectorBuilder.streamTestLimitReached": "Stream testing limit reached. During testing only at max 5 stream slices with 5 pages each will be returned",
```

Comment:
Nitpick: Could we end the 2nd sentence also with a `.`? It feels a bit weird for me having correct punctuation in the beginning of the tooltip, but not in the end.